spec/lib/doi_search_spec.rb

```diff
@@ -1,36 +1,101 @@
-
 describe DoiSearch do
+  let(:other_doi) { '10.1038/psp.2013.66' }
   let(:doi_value) { '10.1016/j.mcn.2012.03.007' }
-  let(:doi_identifier) do
-    FactoryBot.create(:publication_identifier,
-                       identifier_type: 'doi',
-                       identifier_value: doi_value,
-                       identifier_uri: "http://dx.doi.org/#{doi_value}")
-  end
+  let(:doi_identifier) { create(:doi_pub_id, identifier_value: doi_value) }
+  let(:wos_html) { File.read('spec/fixtures/wos_client/medline_encoded_records.html') }
+  let(:wos_records) { WebOfScience::Records.new(encoded_records: wos_html) }
```

Comment:
After the factory changes, it's no longer obvious what the `doi_identifier` contains; the `identifier_uri` is now an "under-the-table" construct (if I understand the changes correctly).

Reply:
Yes, the default is closer to a "good" object, which is the more typical case for consumers of the factory.